### ISSUE TITLE
What is this 3yrs old of a repo

### ISSUE THREAD

**Issue by Sup0rsonic** [openai/grok#11]

**Comment by we-shall** [comment]
I think they renamed their old repo to "grok".

**Comment by brkydnc** [comment]
Yeah, they are obviously trolling.

**Comment by AdrianDittmann** [comment]
Yeah, i was sent this on 𝕏 and decided to come over 😂

**Comment by we-shall** [comment]
I think this can be achieved by changing the OS's system time(to 3 years back) and then committing the code locally. Push the changes to the remote GitHub repo. Yeah?

**Comment by b5i** [comment]
> I think this can be achieved by changing the OS's system time(to 3 years back) and then committing the code locally. Push the changes to the remote GitHub repo. Yeah?

You just have to modify the commit’s date for example by using [redate](https://github.com/PotatoLabs/git-redate).

**Comment by gautamkrishnar** [comment]
Yes, this repo technically existed 3 years ago since the first issue in the repo dates back to  Jan 13, 2022 (#12).  You can't modify a GitHub issue creation timestamp. 

Same for the pull requests. The pull requests (#1) and others indicate that the repo existed for a while. 

I seriously doubt that they did the pre-dating of commits here. I was surprised to see the mention of Grok even in the first Pull request. Good troll btw. Whoever did this deserves a raise.  🤣

**Comment by Yamiqu** [comment]
yeah, 

> This repo technically existed 3 years ago since the first issue in the repo dates back to Jan 13, 2022 (#2). You can't modify a GitHub issue creation timestamp.
> 
> Same for the pull requests. The pull requests (#1) and others indicate that the repo existed for a while.
> 
> I don't think that they did the pre-dating of commits here due to the above pull request. I was surprised to see the mention of Grok even in the first Pull request. Good troll btw. Whoever did this deserves a raise. 🤣
> 
> UPDATE: Wayback machine also has an old snapshot: https://web.archive.org/web/20211204052823/https://github.com/openai/grok

Yes, and I think this repo is actually associated with the paper 
'Grokking: Generalization Beyond Overfitting on Small Algorithmic Datasets' 
published by OpenAI on January 6, 2022 :
https://arxiv.org/abs/2201.02177

**Comment by botserreshef** [comment]
yeah, there are a lot of bugs and problematic code,
I wrote a version of this code with fixes I wanted to add, and also organized the code, not sure if someone is interested...
some functions where not implemented and I did not add them yet
